core/warning/element_reference_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../../spec_helper'
+
+ruby_version_is '2.7' do
+  describe "Warning.[]" do
+    it "returns default values for categories :deprecated and :experimental" do
+      Warning[:deprecated].should == true
```

Comment:
Actually `deprecated` defaults to `false` in `2.7.2` (but it is `true` in 2.7.0&1).
I'd suggest using `ruby_version_is` guards to test that one since it's the big change in 2.7.2:
https://www.ruby-lang.org/en/news/2020/10/02/ruby-2-7-2-released/

Also I just updated the CI to use 2.7.2 so this would actually fail the CI when rebased.